lib/core/src/Cardano/Pool/DB/Sqlite.hs

```diff
@@ -837,6 +852,23 @@ fromPoolProduction (PoolProduction pool slot headerH parentH height) =
         }
     )
 
+mkBlockHeader
+    :: BlockHeader
+    -> TH.BlockHeader
+mkBlockHeader block = TH.BlockHeader
+    { blockSlot = view #slotNo block
+    , blockHeaderHash = BlockId (headerHash block)
+    , blockParentHash = BlockId (parentHeaderHash block)
+    , TH.blockHeight = getQuantity (blockHeight block)
+    }
+
+fromBlockHeaders :: TH.BlockHeader -> BlockHeader
+fromBlockHeaders TH.BlockHeader { .. } =
```

Comment:
Note that, whenever possible, we avoid RecordWildCards and prefer instead NamedFieldPuns. RecordWildCards create implicit let bindings in scope that can be hard to follow, especially in large functions. I don't think it's a problem here because the function is straightforward to follow; yet I'd still favor NamedFieldPuns approach for consistency with the rest of the codebase :+1: